src/test/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParserTest.java

```diff
@@ -397,13 +396,14 @@ public void testArgumentUseClash() {
                 "positional2",
         };
         final FrobnicateArguments fo = new FrobnicateArguments();
-        final CommandLineParser clp = new CommandLineParser(fo);
+        final CommandLineArgumentParser clp = new CommandLineArgumentParser(fo);
         clp.parseArguments(System.err, args);
     }
 
     @Test
     public void testArgumentsFile() throws Exception {
-        final File argumentsFile = BaseTest.createTempFile("clp.", ".arguments");
```

Comment:
it's a shame this isn't available in it's own tiny little artifact.  It would save a lot of annoyance.